src/sqlfluff/rules/L003.py

```diff
@@ -73,8 +73,14 @@ def _eval(self, context: RuleContext) -> List[LintResult]:
           indent meta segment in the previous line.
 
         """
-        return (
+        # NOTE: Until rule consolidation, we're going to filter out
+        # any long line fixes. After that we'll consolidate the two
+        # into one rule.
+        results = (
             ReflowSequence.from_root(context.segment, context.config)
             .reindent()
             .get_results()
         )
+        return [
+            res for res in results if not res.description.startswith("Line is too long")
```

Comment:
Could there be value in formalizing the source / reason for a rule? I.e. something that is guaranteed to be unique and not change such as an `Enum`. In the past, we've occasionally changed descriptions, which would have broken this code in a non-obvious way.

Related: I see code in L016 which does:
```
res.source == "reflow.long_line"
```

Is that doing something similar? Could we use that here?

Reply:
Ah yes - we should do exactly do that. Must have missed this bit. Good catch!